backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -371,6 +458,7 @@ public void send(BacktraceReport report, final OnServerResponseEventListener cal
         if (breadcrumbs != null) {
             breadcrumbs.processReportBreadcrumbs(report);
         }
+        processReportAttachments(report);
```

Comment:
maybe better name can be 'addReportAttachments'?

Reply:
+1